packages/apps/dev-wallet/src/App/layout.tsx

```diff
@@ -1,27 +1,62 @@
-import { Stack, Text } from '@kadena/react-ui';
-import { FC } from 'react';
+import { useNetwork } from '@/modules/network/network.hook';
+import {
+  KadenaLogo,
+  NavHeader,
+  NavHeaderLinkList,
+  NavHeaderSelect,
+  SelectItem,
+  SystemIcon,
+  Text,
+} from '@kadena/react-ui';
+import { FC, useState } from 'react';
 import { Link, Outlet } from 'react-router-dom';
-import { layout } from './layout.css';
 
 export const Layout: FC = () => {
+  const { networks, activeNetwork, updateNetwork } = useNetwork();
+  const [value, setValue] = useState(activeNetwork?.networkId);
```

Comment:
why do we need this state
cant we just use `activeNetwork.networkId`?

Reply:
yes will clear this indeed is redundant